exercises/beer-song/src/test/scala/BeerSongTest.scala

```diff
@@ -0,0 +1,60 @@
+import org.scalatest._
+
+class BeerSongTest extends FunSuite {
+
+  test("first generic verse") {
+    val expected = "99 bottles of beer on the wall, 99 bottles of beer.\nTake one down and pass it around, 98 bottles of beer on the wall.\n"
+    val beerSong = new BeerSong()
+    assert(expected == beerSong.verse(99))
```

Comment:
Same here. The difference is small enough that people won't have much trouble I think